docs/UserGuide.md

```diff
@@ -104,7 +104,7 @@ Adds a job application from a candidate to the application list.
 Format: `add_app e/EMAIL ti/TITLE [s/STATUS]`
 
 * The application added will contain the candidate with the specified email and the job with the specified title.
-* If `STATUS` is omitted, then the `STATUS` of the application will be `PRESCREEN`.
+* There are 5 valid status: `PRESCREEN`, `REJECTED`, `WAITLIST`, `IN_PROGRESS`, and `OFFERED`. If `STATUS` is omitted, then the `STATUS` of the application will be `PRESCREEN`.
```

Comment:
This part may not be needed as its covered in Yan Jie's UG PR where he covered all valid statuses at the start

Reply:
Okie noted